src/main/java/frc/utils/CMDHandler.java

```diff
@@ -0,0 +1,28 @@
+package frc.utils;
+
+import org.littletonrobotics.junction.Logger;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+public class CMDHandler {
```

Comment:
think about adding run java and run python command functions, that only get the class or something like that